pyplanet/apps/contrib/admin/server.py

```diff
@@ -22,7 +25,8 @@ def __init__(self, app):
 			Command(command='setpassword', aliases=['srvpass'], target=self.set_password, perms='admin:password', admin=True).add_param(name='password', required=False),
 			Command(command='setspecpassword', aliases=['spectpass'], target=self.set_spec_password, perms='admin:password', admin=True).add_param(name='password', required=False),
 			Command(command='servername', target=self.set_servername, perms='admin:servername', admin=True).add_param(name='server_name', required=True, nargs='*'),
-			Command(command='mode', target=self.set_mode, perms='admin:mode', admin=True).add_param(name='mode', required=True, nargs='*')
+			Command(command='mode', target=self.set_mode, perms='admin:mode', admin=True).add_param(name='mode', required=True, nargs='*'),
+			Command(command='modesettings', target=self.mode_settings, perms='admin:mode', admin=True).add_param(name='content', required=False, nargs='*')
```

Comment:
Add 2 params, one with nargs=1 (default, so undeclare it) and one with * (and combine it as one string).